src/sentry/utils/email/message_builder.py

```diff
@@ -191,27 +202,38 @@ def get_built_messages(self, to=None, cc=None, bcc=None):
             logger.debug("Did not build any messages, no users to send to.")
         return results
 
-    def format_to(self, to):
+    def format_to(self, to: list[str]) -> str:
         if not to:
             return ""
         if len(to) > MAX_RECIPIENTS:
             to = to[:MAX_RECIPIENTS] + [f"and {len(to[MAX_RECIPIENTS:])} more."]
         return ", ".join(to)
 
-    def send(self, to=None, cc=None, bcc=None, fail_silently=False):
+    def send(
+        self,
+        to: list[str] | None = None,
+        cc: Iterable[str] | None = None,
+        bcc: Iterable[str] | None = None,
+        fail_silently: bool = False,
+    ) -> int:
         return send_messages(
             self.get_built_messages(to, cc=cc, bcc=bcc), fail_silently=fail_silently
         )
 
-    def send_async(self, to=None, cc=None, bcc=None):
+    def send_async(
+        self,
+        to: list[str] | None = None,
+        cc: Iterable[str] | None = None,
+        bcc: Iterable[str] | None = None,
+    ) -> None:
         from sentry.tasks.email import send_email
 
         fmt = options.get("system.logging-format")
         messages = self.get_built_messages(to, cc=cc, bcc=bcc)
-        extra = {"message_type": self.type}
+        extra: MutableMapping[str, str | tuple[str]] = {"message_type": self.type}
```

Comment:
I skimmed around a bit and it looks like the value here should just be `str`, I didn't see where it might be `tuple[str]`?

Reply:
There's a sneaky tuple on line 244:
```py
extra["message_to"] = (self.format_to(message.to),)
```